trunk/src/graphics/paraview/geom2vtk.F90

```diff
@@ -135,6 +147,32 @@ subroutine geom2vtk(Sname,Sfile, IcE,IcN,IcP)
 !..Step 3 : Elements
 !
    ice_subd=0; icn_subd=0
+!..Additional Computational for computing the total number of elements (including subelements: vlevel > 0))
+!..VTU Format needs this information apriori as VTU needs headers with this information before appending the data.   
+   if(VIS_VTU) then
+      if(SECOND_ORDER_VIS) then
+         allocate(ELEM_TYPES(NRELES))
```

Comment:
`ELEM_TYPES` is being allocated here but never deallocated in this routine or a corresponding system routine. This will lead to issues (either a user trying to deallocate the array when it's not allocated, or the user simply forgetting deallocation so that this memory is never deallocated in repeated calls to `geom2vtk`. As an example, the default `paraview_driver` in `src/graphics/paraview_driver` never deallocates this.

Reply:
I will deallocate ELEM_TYPES  within geom2vtk.F90. I think then It should work.

Reply:
As long as it's only needed within `geom2vtk`, that sounds like the right solution.